Sources/AppcuesKit/Presentation/Public/AppcuesViewElement.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  AppcuesViewElement.swift
+//  AppcuesKit
+//
+//  Created by James Ellis on 3/22/23.
+//  Copyright © 2023 Appcues. All rights reserved.
+//
+
+import Foundation
+
+/// Represents a view in the layout hierarchy of the application.
+///
+/// The view information provided in this structure provides screen capture metadata to Appcues
+/// servers that can be used for element targeted experiences. The layout information is used by the
+/// Appcues Mobile Builder to select and target UI elements, and used by the Appcues iOS SDK to
+/// position experience content relative to targeted elements.
+@objc
+public class AppcuesViewElement: NSObject, Encodable {
+    /// Auto-generated unique ID for the view.
+    let id = UUID().appcuesFormatted
+
+    /// The x-coordinate for view position, with origin in the upper-left corner.
+    let x: CGFloat
+
+    /// The y-coordinate for view position, with origin is in the upper-left corner.
+    let y: CGFloat
```

Comment:
Perhaps specify these values are in screen coordinates, not relative to the parent?